CIP-0001/README.md

```diff
@@ -249,7 +249,7 @@ Registered categories for explicitly enlisted projects are otherwise listed belo
 Category | Description
 ---      | ---
 Plutus   | Changes or additions to Plutus, following the process described in [CIP-0035][]
-Ledger   | For proposals regarding the Cardano ledger, following the process described in [CIP-0084][https://github.com/cardano-foundation/CIPs/pull/456]
+Ledger   | For proposals regarding the Cardano ledger, following the process described in [CIP-0084][]
```

Comment:
you will need to add the CIP-0084 link to the references (past line 442) :)

Reply:
fixed as of https://github.com/cardano-foundation/CIPs/pull/730/commits/0708c7e5905c96105f2a6de216519eafe076f930

Reply:
:)